src/ImageSharp/Formats/Tiff/ITiffEncoderOptions.cs

```diff
@@ -37,5 +37,15 @@ internal interface ITiffEncoderOptions
         /// Gets the quantizer for creating a color palette image.
         /// </summary>
         IQuantizer Quantizer { get; }
+
+        /// <summary>
+        /// Gets the pixel storage method.
+        /// </summary>
+        TiffEncoderPixelStorageMethod PixelStorageMethod { get; }
+
+        /// <summary>
+        /// Gets the maximum size of strip (bytes).
+        /// </summary>
+        int MaxStripBytes { get; }
```

Comment:
this is a limit of one strip length (the field also can be used for tiles)

Reply:
this field can also be renamed 😄

Reply:
i think `MaxStripBytes` is an ok name, its clear what it does.

Reply:
Is that interface public? I’m curious as to what the property means as a configuration value to the average user

Reply:
The interface is `internal`. 
I looked `StripByteCounts` on different `tiff` files and watched that they have different lengths, not only 8Kb, in some cases it reaches 1Mb. So I figured that the user might want to set the strip size.

Reply:
Pardon my ignorance but what do the different strip count length mean in terms of encoding?

Reply:
This is the size of encoder buffer, the image is divided into `strips` and encoding one by one, so there is no need to allocate additional large buffers. It seems that there is no single agreement, there is only a recommendation about `8Kb` (also I read about `64Kb`), I think most users will be satisfied with default size. But maybe someone wants to adjust the strip size himself. Maybe @brianpopow will give more complete information?

Reply:
frankly, for modern computers, I think it won't be a problem using `1Mb` buffers

Reply:
Thanks... I guess we can determine a sensible default by benchmarking.